static/app/gettingStartedDocs/java/java.tsx

```diff
@@ -86,8 +93,8 @@ sentry {
   // code as part of your stack traces in Sentry.
   includeSourceContext = true
 
-  org = "___ORG_SLUG___"
-  projectName = "___PROJECT_SLUG___"
+  org = "${organizationSlug}"
+  projectName = "${projectSlug}"
```

Comment:
Should this be inside backticks to work?

Reply:
The whole assignment is inside a snippet / inside backticks

Reply:
gotcha 👍